core-skills/react/README.md

```diff
@@ -80,11 +81,12 @@ During the assessment, the reviewer will go through your application with you, a
 #### Application - Mark scheme
 
 - The candidate has created an application which contains a component:
-    - That uses makes use of the `key` prop
-    - Without any props of state
+    - That makes use of the `key` prop
     - Which makes use of a function passed in as a prop
     - Which renders differently based on state
+    - Without any state
     - Which changes state in response to an onClick event
+    - Which makes use of `useState()` within a functional component
```

Comment:
Is there any requirement to create a class based component, or is it encouraged to use entirely functional components?

Reply:
I think we're looking for a mix of the two to show competence? Maybe we can clarify slightly that we want both? (/are we looking for both?)